witchcraft-server/src/debug/mod.rs

```diff
@@ -42,32 +43,38 @@ pub trait Diagnostic {
 }
 
 pub struct DiagnosticRegistry {
```

Comment:
This'll need docs now if it's being made public.

Reply:
Done!